hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseTokenHandler.java

```diff
@@ -429,31 +429,14 @@ public static boolean isExpiryOnlyUpdateOp(@NonNull final TokenUpdateTransaction
     }
 
     /**
-     * Check if TokenUpdateOp wants to update only some of the low priority keys or the metadata field
-     * low priority keys are -> wipeKey, kycKey, supplyKey, freezeKey, feeScheduleKey, pauseKey or metadataKey
-     */
-    public static boolean noOtherFieldThanLowPriorityKeyOrMetadataWillBeUpdated(
-            @NonNull final TokenUpdateTransactionBody op) {
-        return !((op.symbol() != null && !op.symbol().isEmpty())
-                || (op.name() != null && !op.name().isEmpty())
-                || op.hasTreasury()
-                || op.hasAdminKey()
-                || op.hasAutoRenewAccount()
-                || op.hasAutoRenewPeriod()
-                || op.hasMemo()
-                || op.hasExpiry());
-    }
-
-    /**
-     * Check if a given token already has some of the low priority keys
+     * Checks if the given op updates any of the non-key token properties that can only be
+     * changed given the admin key signature.
+     *
+     * @param op the token update op to check
+     * @return true if it requires admin key signature
      */
-    public static boolean hasAlreadySomeNonAdminKeys(@NonNull final Token token) {
-        for (final var nonAdminKey : NON_ADMIN_TOKEN_KEYS) {
-            if (nonAdminKey.isPresentInitially(token)) {
-                return true;
-            }
-        }
-        return false;
+    protected static boolean updatesAdminOnlyNonKeyTokenProperty(@NonNull final TokenUpdateTransactionBody op) {
+        return !op.symbol().isEmpty() || !op.name().isEmpty() || op.hasAutoRenewPeriod() || op.hasMemo();
```

Comment:
Its better to add a comment that we are already checking autoRenew Id and treasury. That's why we are not including here even though the name of the function suggests all fields except non-key token property